api/jobs.go

```diff
@@ -266,9 +266,9 @@ func (j *Jobs) ScaleStatus(jobID string, q *QueryOptions) (*JobScaleStatusRespon
 
 // Versions is used to retrieve all versions of a particular job given its
 // unique ID.
-func (j *Jobs) Versions(jobID string, diffs bool, q *QueryOptions) ([]*Job, []*JobDiff, *QueryMeta, error) {
+func (j *Jobs) Versions(jobID string, diffs bool, diffTag string, diffVersion string, q *QueryOptions) ([]*Job, []*JobDiff, *QueryMeta, error) {
 	var resp JobVersionsResponse
-	qm, err := j.client.query(fmt.Sprintf("/v1/job/%s/versions?diffs=%v", url.PathEscape(jobID), diffs), &resp, q)
+	qm, err := j.client.query(fmt.Sprintf("/v1/job/%s/versions?diffs=%t&diff_tag=%s&diff_version=%s", url.PathEscape(jobID), diffs, diffTag, diffVersion), &resp, q)
```

Comment:
I ended up making my options params into a `url.Values{}` which I then `.encode()` when appending into the URL string. I dug into the `url.Values{}.encode()`  code a bit and it looks like it `QueryEscape()`s, so I think this will be solved that way (I suspect it wouldn't be too bad for me to add a possibly-redundant QueryEscape here but doesn't seem necessary now)